mis_builder/models/mis_report_instance.py

```diff
@@ -801,6 +801,17 @@ def _add_column_move_lines(self, aep, kpi_matrix, period, label, description):
             period._get_additional_move_line_filter(),
             period.source_aml_model_name,
         )
+        annotations = self.env["mis.report.instance.annotation"].search(
+            [
+                ("period_id", "=", period.id),
+            ]
+        )
+
+        note_by_subkpi_id = {
+            (note.kpi_id.id, note.subkpi_id.id): note.note for note in annotations
+        }
+        locals_dict = {"notes": note_by_subkpi_id}
```

Comment:
I need to look closer, but locals_dict is used as the evaluation context for expressions. I don't think it is useful to add notes in that context